python/setup.py

```diff
@@ -21,6 +21,7 @@
 import versioneer
 from distutils.sysconfig import get_python_lib
 import sys
+import numpy
```

Comment:
This is what I want to avoid with my suggestions about not using the numpy cython api. This has caused ABI headaches in the past, so if possible to avoid would be nice.